app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapFragment.java

```diff
@@ -432,7 +432,6 @@ public boolean isNetworkConnectionEstablished() {
     public void populatePlaces(LatLng curLatLng) {
         final Observable<MapController.ExplorePlacesInfo> nearbyPlacesInfoObservable;
         if (curLatLng == null) {
-            checkPermissionsAndPerformAction();
```

Comment:
Is this removal really needed to fix the issue #5490? 🤔

Reply:
This removal was necessary as if this would be there, then the app would ask for permission as soon as the map resumes, and I have changed the logic for that, so it's of no use here.